src/app/projects/project-list/project-list.component.html

```diff
@@ -38,12 +33,9 @@
         </tr>
       </ng-template>
       <ng-template pTemplate="body" let-project>
-        <wh-project-list-item
-          [project]="project"
-          (refreshed)="onProjectRefreshed($event)"
-          (favoriteClick)="initFavoriteProjects()"
-          role="button"
-          (click)="onProjectRowClicked($event, project)"></wh-project-list-item>
+        <wh-project-list-item [project]="project" (refreshed)="onProjectRefreshed($event)"
+          (favoriteClick)="initFavoriteProjects()" role="button" (click)="onProjectRowClicked($event, project)">
```

Comment:
This appears to contain a formatting change. To me the previous format multi-lined attributes is easier to read. This should probably be decided on a repo as a whole basis and then added as formatting rules and appropriately linted.

Reply:
I agree that having it on multiple lines makes it easier to read.

> This should probably be decided on a repo as a whole basis and then added as formatting rules and appropriately linted.

That sound like a good idea. For this PR I'll see if I can change my settings to match the old format.